Podfile.lock

```diff
@@ -196,4 +196,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 41411a660d9001775e42d2a8861853f682217eb8
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.9.3
```

Comment:
If you change this back you should be able to get further in the CI process so that you can actually see your tests run. Right now there is a mismatch between what you have here and what the build is expecting